column/utils.py

```diff
@@ -35,3 +36,65 @@ def vault_encrypt(value):
         _get_vault_password_file(), dataloader.DataLoader())
     this_vault = vault.VaultLib(vault_password)
     return this_vault.encrypt(value)
+
+
+def reset_play_context():
```

Comment:
These settings make me very uneasy. We setting all kinds of things and overriding some defaults from ansible.cfg constants.